bin_tests/tests/test_the_tests.rs

```diff
@@ -14,6 +14,7 @@ fn test_the_tests_debug() {
 }
 
 #[test]
+#[ignore] // This test is slow, only run it if explicitly opted in
 #[cfg_attr(miri, ignore)]
```

Comment:
If this test is ignore, can `#[cfg_attr(miri, ignore)]` this be removed?

Reply:
done